Freedrenot with KGSL will never support WSI, so do not compile these files.

--- a/src/freedreno/vulkan/meson.build
+++ b/src/freedreno/vulkan/meson.build
@@ -70,7 +70,7 @@
   tu_wsi = true
 endif
 
-if system_has_kms_drm and not with_platform_android
+if system_has_kms_drm and not with_platform_android and not with_freedreno_kgsl
   libtu_files += files('tu_wsi_display.c')
   tu_wsi = true
 endif
